requirements.dev.txt

```diff
@@ -5,7 +5,7 @@ mypy==1.5.1
 # Same version from pghoard
 pylint==2.17.4
 botocore
-botocore-stubs[essential]
+botocore-stubs
```

Comment:
nit (non-blocking): this change is unrelated to pydantic, should probably go into separate commit